examples/c/ex_all.c

```diff
@@ -376,7 +376,12 @@ cursor_ops(WT_SESSION *session)
         error_check(
           session->open_cursor(session, "table:mytable", NULL, "overwrite=false", &cursor));
         cursor->set_key(cursor, key);
-        error_check(cursor->remove(cursor));
+        if ((ret = cursor->remove(cursor)) != 0) {
+            fprintf(stderr, "cursor.remove: %s\n", wiredtiger_strerror(ret));
+            /* We expect to get a WT_NOTFOUND error if we try to remove a record that does not
+             * exist. */
+            WT_RET_NOTFOUND_OK(ret);
```

Comment:
We don't want to put `WT_RET_NOFOUND_OK` in the documentation, that's an internal macro, it's likely to be confusing in the documentation.  Maybe this (and the next few changes in this file?), could be something like this:
```
index 314bb12b0..7f06076fe 100644
--- a/examples/c/ex_all.c
+++ b/examples/c/ex_all.c
@@ -376,12 +376,11 @@ cursor_ops(WT_SESSION *session)
         error_check(
           session->open_cursor(session, "table:mytable", NULL, "overwrite=false", &cursor));
         cursor->set_key(cursor, key);
-        if ((ret = cursor->remove(cursor)) != 0) {
+       /* We expect to get a WT_NOTFOUND error if we try to remove a record that does not exist. */
+        if ((ret = cursor->remove(cursor)) == WT_NOTFOUND)
             fprintf(stderr, "cursor.remove: %s\n", wiredtiger_strerror(ret));
-            /* We expect to get a WT_NOTFOUND error if we try to remove a record that does not
-             * exist. */
-            WT_RET_NOTFOUND_OK(ret);
-        }
+       else
+            error_check(ret);
         /*! [Remove a record and fail if DNE] */
     }
```